### PR TITLE
PWN-7376 - tokens to claim created in TokenAdapter

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7376

## Description of Work

PWN-7376 - tokens to claim created in TokenAdapter

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/main/adapter/TokenToClaimViewHolder.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    private fun setClaimButtonEnabled(isEnabled: Boolean) {
+        binding.buttonClaim.isEnabled = isEnabled
+        with(binding.buttonClaim) {
+            if (isEnabled) {
+                setText(R.string.claim_button_text)
+                setTextColor(getColor(R.color.text_snow))
+                setBackgroundColor(getColor(R.color.bg_night))
+            } else {
+                setText(R.string.claiming_button_text)
+                setTextColor(getColor(R.color.text_mountain))
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/main/adapter/TokenToClaimViewHolder.kt:35]: i think we have setTextColorRes

File: app/src/main/res/layout/item_token_to_claim.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            android:layout_height="48dp"
+            android:layout_marginVertical="16dp"
+            app:cornerRadius="12dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:src="@tools:sample/avatars" />
+
+        <ImageView
+            android:id="@+id/wrappedImageView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_token_to_claim.xml:32]: imageViewWrapped

File: app/src/main/res/layout/item_token_to_claim.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            android:layout_marginBottom="-4dp"
+            android:src="@drawable/ic_wrapped"
+            android:visibility="gone"
+            app:layout_constraintBottom_toBottomOf="@id/imageViewToken"
+            app:layout_constraintEnd_toEndOf="@id/imageViewToken"
+            tools:ignore="ContentDescription"
+            tools:visibility="visible" />
+
+        <TextView
+            android:id="@+id/textViewName"
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_token_to_claim.xml:44]: textViewTokenName

File: app/src/main/res/layout/item_token_to_claim.xml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            android:maxLines="1"
+            android:textAppearance="@style/UiKit.TextAppearance.Regular.Text2"
+            app:layout_constraintBottom_toTopOf="@id/textViewTotal"
+            app:layout_constraintEnd_toStartOf="@id/buttonClaim"
+            app:layout_constraintStart_toEndOf="@id/imageViewToken"
+            app:layout_constraintTop_toTopOf="@id/imageViewToken"
+            tools:text="Wrapped Ethereum" />
+
+        <TextView
+            android:id="@+id/textViewTotal"
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_token_to_claim.xml:58]: textViewTokenTotal

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/home/model/HomeElementItem.kt [hunk 5]
```diff
@@ -3,6 +3,7 @@ package org.p2p.wallet.home.model
 import org.p2p.core.token.Token
 
 sealed class HomeElementItem {
+    data class Claim(val claimStatus: Boolean) : HomeElementItem()
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/home/model/HomeElementItem.kt:6]: `isClaimEnabled`

**Review by 1sakov — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

**Review by NikolaiEmelianov — APPROVED** [review]

**Pull request merged**